plugins/rtmp-services/data/services.json

```diff
@@ -1,5 +1,5 @@
 {
-    "format_version": 1,
+    "format_version": 2,
```

Comment:
This is most likely due to the addition of the ftl type of service not being backwards compatible and thus making this a new version of the format. It also means that older versions of obs won't load the new services file as it could cause issues.